theme/lumo/vaadin-message-styles.js

```diff
@@ -3,40 +3,36 @@ import '@vaadin/vaadin-lumo-styles/color.js';
 import '@vaadin/vaadin-lumo-styles/sizing.js';
 import '@vaadin/vaadin-lumo-styles/spacing.js';
 import '@vaadin/vaadin-lumo-styles/style.js';
+import '@vaadin/vaadin-lumo-styles/typography.js';
 import '@vaadin/vaadin-avatar/theme/lumo/vaadin-avatar.js';
 
 registerStyles(
   'vaadin-message',
   css`
     :host {
-      padding: var(--lumo-space-s);
-    }
-
-    :host {
+      color: var(--lumo-body-text-color);
       font-family: var(--lumo-font-family);
       font-size: var(--lumo-font-size-m);
+      line-height: var(--lumo-line-height-m);
+      padding: var(--lumo-space-s) var(--lumo-space-m);
+      -moz-osx-font-smoothing: grayscale;
+      -webkit-font-smoothing: antialiased;
+      -webkit-text-size-adjust: 100%;
     }
 
-    .vaadin-message-wrapper {
-      padding-left: var(--lumo-space-s);
-      padding-right: 0;
-    }
-
-    :host([dir='rtl']) .vaadin-message-wrapper {
-      padding-left: 0;
-      padding-right: var(--lumo-space-s);
-    }
-
-    .vaadin-message-header {
-      padding-bottom: var(--lumo-space-s);
+    vaadin-avatar {
+      margin-inline-end: var(--lumo-space-s);
```

Comment:
Note that this doesn't work with IE11. I also implemented this first with inline styles (padding-inline-styles) with a special rule for IE specifically. It was there agreed upon that we should use the margin-left / -right combo instead of combination with inline styles and media rules for IE11 with margin-left/right.

Please change back to margin-left and margin right, separately for LTR and RTL.

See the discussion here. https://github.com/vaadin/vaadin-messages/pull/6#discussion_r568432738

Reply:
Gotcha, will do.